### DISCUSSION TITLE
Tailwind not working

### DISCUSSION THREAD

**Discussion by McCarthyCharles233** [tailwindlabs/tailwindcss#9900] [Help]
I've installed tailwind CSS into a project. Installation was done successfully but styles are not reflecting on HTML page. Please I need help in how to fix this

**Comment by wongjn** [comment]
We're going to need more information to go on to troubleshoot your issue! Somethings to check:
- Ensure you've followed the [installation steps](https://tailwindcss.com/docs/installation) most appropriate for your project.
- Ensure your PostCSS configuration has the Tailwind plugin in the set of plugins (if applicable).
- [Ensure your `content` file globs in your Tailwind configuration sufficiently covers your source files](https://tailwindcss.com/docs/content-configuration).

If you still cannot get it working after the above steps, please consider providing a remotely-hosted git repository so that we can take a closer look.

**Comment by DoudGaya** [comment]
If you provide an image of your HTML and tailwind-generated CSS file, it'll be a lot easier for us to help. a quick guideline though is to install tailwind using this command 
`
npm install -D tailwindcss 
npx tailwindcss init
`

then inside `tailwind.config.js` make sure the content looks something like this
` content: ["./src/**/*.{html, js}"],`

create a folder called `src` and add a CSS file in it with the name  `input.css`. Then add these 3 lines to the file 

`
@tailwind base;
@tailwind components;
@tailwind utilities;
`
then you run this command. It'll take a file you created

`npx tailwindcss -i ./src/input.css -o ./dist/output.css --watch`

then in your HTML, this is how you reference it in your header 

`
<link href="/dist/output.css" rel="stylesheet">
`

it should work. when you set it up this way
